vulcan/_vulcan.py

```diff
@@ -121,3 +122,32 @@ def get_homework(self, date=None):
             :class:`vulcan._homework.Homework`
         """
         return Homework.get(self._api, date)
+
+    def get_message(self, date=None):
```

Comment:
You've just changed it again to `get_message`. Don't forget to pull after you make any changes on GitHub!
```suggestion
    def get_messages(self, date=None):
```

Reply:
Yes, i don't know why but i did this